doc/user/isisd.rst

```diff
@@ -337,6 +337,15 @@ ISIS interface
    Set PSNP interval in seconds globally, for an area (level-1) or a domain
    (level-2).
 
+.. index:: isis three-way-handshake
+.. clicmd:: isis three-way-handshake
+
+.. index:: no isis three-way-handshake
+.. clicmd:: no isis three-way-handshake
+
+   Enable or disable RFC5303 Three-Way Handshake for P2P adjacencies.
```

Comment:
If you want, Sphinx will automatically generate a link to the RFC if you use the following markup:

   `rfc`:5303